robots/EDUCATOR/color.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+"""Make robot say whatever color it observes with the color sensor."""
+
+from ev3dev.sensor.lego import ColorSensor
```

Comment:
Can you import from `ev3dev.sensor` here instead? Frankly, I think we should make the `lego` module "python private" and only advertise the `ev3dev.sensor` module. I don't see a reason that a user would want more detail.